### DISCUSSION TITLE
Is it possible to use Redux?

### DISCUSSION THREAD

**Discussion by DonikaV** [inertiajs/inertia#979] [Ideas]
I am going to create admin dashboard Laravel + Inertia + React, but all dashboards themes are comes with Redux. 
Is it possible to use redux?

**Comment by ajnsn** [comment]
Hey @DonikaV 
one of the benefits of Inertia is that you keep all state server-side. So you should try to avoid any client state. But sure, it's possible to keep client state, even when doing an Inertia round trip by [preserving state](https://inertiajs.com/links#preserve-state).

**Reply by DonikaV** [reply]
> Hey @DonikaV 
> 
> one of the benefits of Inertia is that you keep all state server-side. So you should try to avoid any client state. But sure, it's possible to keep client state, even when doing an Inertia round trip by [preserving state](https://inertiajs.com/links#preserve-state).

Thanks!
